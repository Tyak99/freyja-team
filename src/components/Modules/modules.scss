$base-color: #5f3518;
$white: #fff;
$text-color: #3f3d3c;
$module-card-shadow: rgba(54, 52, 52, 0.4);

.m-container {
  width: 100%;
}

.m-container div .card{
  margin-bottom: 20px;
  position: unset;
  height: 22rem;
  width: 19rem;
  box-sizing: border-box;
  box-shadow: 10px 10px 10px $module-card-shadow;
  cursor: pointer;
}

.m-intro {
  width: 60%;
  padding-left: 4rem;
  margin: 0 auto 4rem;
}
.m-col-adjust {
  margin-bottom: 15px;
}

.m-container div .card:hover {
  border: 2px $base-color solid;
}
.t-link {
  position: absolute;
  bottom: 20px;
  left: 10px;
}
.s-m-link {
  padding: 4rem .4rem;
  margin: 4rem 0
}
.take-course{
  margin: 5rem auto 10rem;
  width: 10rem;
}
.take-course .btn {
  background: $white;
  color: $text-color;
  border: 2px solid $base-color;
  font-size: 1rem;
  border-radius: 2px;
}
.take-course .btn:hover {
  background: $base-color;
  box-shadow: 0 0 4px rgba(54, 52, 52, 0.4);
  color: $white;
}
.m-c-title .card-title {
  font-weight: bold;
  font-size: 16px;
  text-align: left;
}
.t-n-text{
  color: #272525;
}
.t-d-text {
  color: #666262;
}
.m-c-title .card-subtitle{
  font-weight: 450;
  color: #000
}
 .m-c-title .card-title{
  margin-bottom: 2rem;
}
a.m-card-link {
  text-decoration: none;
}