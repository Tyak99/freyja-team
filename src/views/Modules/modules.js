import React from 'react';
import { Link } from 'react-router-dom';

const Home = () => (
  <div>
    <h2> This is the Module pPage</h2>
    <Link to="/login"> Go to Login Page </Link>
    <br />
    <Link to="/signup"> Go to Signup Page </Link>
  </div>
);

export default Home;
