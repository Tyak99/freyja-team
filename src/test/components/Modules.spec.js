import React from 'react';
import { shallow } from 'enzyme';
import Modules from '../../components/Modules/Modules';


describe('component: Modules', () => {
  const wrapper = shallow(<Modules />);
  it('should render correctly', (done) => {
    expect(wrapper.find('div.m-container').exists()).toBe(true);
    done();
  });
  test('should render modules component correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
