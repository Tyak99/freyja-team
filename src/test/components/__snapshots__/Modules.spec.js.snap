// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component: Modules should render modules component correctly 1`] = `ShallowWrapper {}`;
